nautobot/dcim/tests/test_models.py

```diff
@@ -3497,7 +3497,7 @@ def test_modifying_vdc_device_not_allowed(self):
         self.assertIn("Virtual Device Context's device cannot be changed once created", str(err.exception))
 
 
-class ModuleFamilyTestCase(TestCase):
+class ModuleFamilyTestCase(ModelTestCases.BaseModelTestCase):
```

Comment:
Will need to add `model = ModuleFamily` as a class attribute to make `BaseModelTestCase` happy.